server.js

```diff
@@ -1,33 +1,94 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import data from "./data/netflix-titles.json";
+
+//I've left all coments so that I can go back to this project and read them if necessary in the future.
+
+// Defines the port the app will run on. Defaults to 8080, but can be
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
+
+const listEndpoints = require("express-list-endpoints");
+
+app.get("/", (req, res) => {
+  res.send(listEndpoints(app));
+});
+
+app.get("/content", (req, res) => {
+  // Sorts based on queries
+  const { page, actor, type, title, year, country, director } = req.query;
+  let dataList = data;
+
+  if (actor) {
+    dataList = dataList.filter((item) =>
+      item.cast.toLowerCase().includes(actor.toLowerCase())
+    );
+  }
+  if (type) {
+    dataList = dataList.filter((item) =>
+      item.type.toLowerCase().includes(type.toLowerCase())
+    );
+  }
+  if (title) {
+    dataList = dataList.filter((item) =>
+      item.title.toString().toLowerCase().includes(title.toLowerCase())
+    );
+  }
+
+  if (year) {
+    dataList = dataList.filter((item) =>
+      item.release_year.toString().toLowerCase().includes(year.toLowerCase())
+    );
+  }
+
+  if (country) {
+    dataList = dataList.filter((item) =>
+      item.country.toString().toLowerCase().includes(country.toLowerCase())
+    );
+  }
+
+  if (director) {
+    dataList = dataList.filter((item) =>
+      item.director.toString().toLowerCase().includes(director.toLowerCase())
+    );
+  }
```

Comment:
This code is so clear.  Naming is simple and effective. 
The structure is easy on the eyes.  See my comment about a reusable function above ☝️